### ISSUE TITLE
tests: PresignedPostPolicy test should perform post operation

### ISSUE THREAD

**Issue by harshavardhana** [minio/minio-go#753]
Currently the presignedPostPolicy is not performing the POST operation after generating the relevant policy. 

```go
        if err := policy.SetContentLengthRange(1024*1024, 1024); err == nil {
                log.Fatalf("Error: %s", err)
        }

        policy.SetBucket(bucketName)
        policy.SetKey(objectName)
        policy.SetExpires(time.Now().UTC().AddDate(0, 0, 10)) // expires in 10 days
        policy.SetContentType("image/png")
        policy.SetContentLengthRange(1024, 1024*1024)

        _, _, err = c.PresignedPostPolicy(policy)
        if err != nil {
                log.Fatal("Error:", err)
        }

        policy = minio.NewPostPolicy()

        // Remove all objects and buckets
        err = c.RemoveObject(bucketName, objectName)
        if err != nil {
                log.Fatal("Error:", err)
        }

        err = c.RemoveBucket(bucketName)
        if err != nil {
                log.Fatal("Error:", err)
        }
```

**Comment by nitisht** [comment]
will be fixed by https://github.com/minio/minio-go/pull/840

**Comment by nitisht** [comment]
fixed by #840
